http.go

```diff
@@ -70,6 +70,10 @@ func (s *Server) ServeHTTPS() {
 		log.Fatalf("FATAL: loading tls config (%s, %s) failed - %s", s.Opts.TLSCertFile, s.Opts.TLSKeyFile, err)
 	}
 
+	if len(s.Opts.TLSClientCAFiles) > 0 {
+		config.ClientAuth = tls.RequestClientCert
```

Comment:
Actually I think coincidentally we get the best of both worlds.  Router doesn't allow client, so you don't get prompted.  Internal does.